### PR TITLE
Counter pane and sprintplans

### PR DESCRIPTION
**Pull request by Markvanderuit**
Adds sprint plans and retrospects for this week to documentation folder.
Changes counter pane to now show a list of numbers.

### CODE REVIEW

**Comment by mennoo1996** [comment]
Seems fine to me. Good work!

File: src/main/java/gui/centerarea/CounterGridPane.java [hunk 1]
```diff
@@ -9,16 +12,64 @@
 public class CounterGridPane extends ScrollableGridPane {
 
     /**
+     * Styling string here is temporary, until my (Mark's) big CSS branch is ready.
+     * It has separate CSS files for all stylized elements, but I'm not merging it
+     * until its done.
+     */
+
```
Comment (bart-de-jonge) [inline src/main/java/gui/centerarea/CounterGridPane.java]: unnecessary white line

File: src/main/java/gui/centerarea/CounterGridPane.java [hunk 2]
```diff
@@ -9,16 +12,64 @@
...
+        // set actual vertical constraints to grid.
+        for (int i = 0; i < numberOfCounts; i++) {
+            RowConstraints rc = new RowConstraints();
+            rc.setMinHeight(verticalElementSize);
+            rc.setPercentHeight(100.0 / numberOfCounts);
+            rc.setVgrow(Priority.ALWAYS);
+            this.getRowConstraints().add(rc);
+        }
+
```
Comment (bart-de-jonge) [inline src/main/java/gui/centerarea/CounterGridPane.java]: white line

File: src/main/java/gui/centerarea/CounterGridPane.java [hunk 3]
```diff
@@ -9,16 +12,64 @@
...
+     * @param verticalElementSize size of usual vertical elements.
+     */
+    private void initGridNumbers(int numberOfCounts, int verticalElementSize) {
+        for (int i = 4; i < numberOfCounts; i += 4) {
+            Label label = new Label(Integer.toString(i));
+            label.setStyle(labelStyle);
+            setHalignment(label, HPos.RIGHT);
+            this.add(label, 0, i);
+        }
     }
```
Comment (bart-de-jonge) [inline src/main/java/gui/centerarea/CounterGridPane.java]: white line

**Comment by bart-de-jonge** [comment]
2 small comments, I will fix them since you are unaivable and then merge

**Pull request merged**